spec/models/course_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'rails_helper'
+
+RSpec.describe Course, type: :model do
+  subject { Course.new(course_type: 'Pilot Courses', info: 'Pilot course category') }
+
+  before { subject.save }
+
+  it 'course type should be present' do
+    subject.course_type = nil
+    expect(subject).to_not be_valid
+  end
+
+  it 'course type length should be >= 3' do
+    subject.course_type = 'Pc'
+    expect(subject).to_not be_valid
+  end
+
+  it 'course type should be unique & not be case sensitive' do
+    subject_two = Course.new(course_type: 'pilot courses', info: 'pilot course category')
+    expect(subject_two).to_not be_valid
+  end
+
+  it 'course info should be present' do
+    subject.info = nil
+    expect(subject).to_not be_valid
+  end
+
+  it 'course type length should be >= 10' do
+    subject.info = 'Pc'
+    expect(subject).to_not be_valid
```

Comment:
It's great that you've done tests but all I see are negative tests - when you expect it to fail. What about when you expect them to be successful. You'd be surprised how many times in the real world something you expect to work does not/ Doing positive tests is also important. It'd save you time while developing and help you catch bugs before they get too messy.

Reply:
Do above for all tests ⬆️

Reply:
Thanks for that. Not enough but I added some positive tests to all models.